installer/scripts/omsadmin.sh

```diff
@@ -353,6 +353,15 @@ onboard_scom()
     #Always register SCOM as secondary Workspace
     echo "SCOM Workspace" > $CONF_DIR/.multihoming_marker
     configure_logrotate
+
+    #Open port 1270 if not already open
+    IS_PORT_OPEN=`cat $OMI_CONF_FILE | grep httpsport | grep 1270 | wc -l`
```

Comment:
Also, wc is not a Linux command we've run during installation before. I would rather we simply evaluate whether `cat $OMI_CONF_FILE | grep httpsport | grep 1270` returns 0 or 1.

It may seem like something the customer should have, but I've seen errors recently with our install because the customer didn't have tar installed :)